tuna/miopen/fin_class.py

```diff
@@ -622,6 +667,48 @@ def process_fdb_w_kernels(self,
 
     return status
 
+  def add_sql_objs(self, session, obj_list):
+    """add sql objects to the table"""
+    self.logger.info("adding pending #objects: %s", len(obj_list))
+    for sql_obj in obj_list:
+      session.add(sql_obj)
+    session.commit()
```

Comment:
could use a try catch in case we run into more db contention

Reply:
this is executed inside a session_retry, see 695